packages/core/src/types/config-root.ts

```diff
@@ -0,0 +1,376 @@
+import { NodeType } from '.';
```

Comment:
I suggest renaming this file into **external-config-schemas.ts** or something like that.